bin/pos-cli-modules-pull.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+const program = require('commander'),
+  ora = require('ora'),
+  Gateway = require('../lib/proxy'),
+  logger = require('../lib/logger'),
+  fetchAuthData = require('../lib/settings').fetchSettings,
+  downloadFile = require('../lib/downloadFile'),
+  waitForStatus = require('../lib/data/waitForStatus');
+const spinner = ora({ text: 'Exporting', stream: process.stdout, spinner: 'bouncingBar' });
+
+program
+  .name('pos-cli modules pull')
+  .arguments('[module]', 'module name to pull')
+  .arguments('[environment]', 'name of the environment. Example: staging')
```

Comment:
Probably order needs to change if https://github.com/mdyd-dev/pos-cli/pull/360/files#r394375563 is accepted